fatal error: stm32_def.h: No such file or directory

Hi there! Really excited to use this library in PlatformIO. However, with a simple project I'm not able to build. Here's my platformio.ini

```
[env:bluepill_f103c8]
platform = ststm32
board = bluepill_f103c8
framework = arduino
debug_tool = stlink-v2
build_flags = -std=gnu++11 -MMD -DCONFIG_MAPLE_MINI_NO_DISABLE_DEBUG=1 -DSERIAL_USB 
build_unflags = -lc -lstdc++ -lnosys

lib_deps =
  Nanopb
  PacketSerial
  STM32duino FreeRTOS
```

When I try to build, I get this error while building FreeRTOS

```
Compiling .pioenvs/bluepill_f103c8/lib823/STM32duino FreeRTOS_ID2093/FreeRTOS/Source/event_groups.o
In file included from .piolibdeps/STM32duino FreeRTOS_ID2093/src/FreeRTOS/Source/include/portable.h:94:0,
from .piolibdeps/STM32duino FreeRTOS_ID2093/src/FreeRTOS/Source/include/FreeRTOS.h:104,
from .piolibdeps/STM32duino FreeRTOS_ID2093/src/FreeRTOS.h:9,
from .piolibdeps/STM32duino FreeRTOS_ID2093/src/FreeRTOS/Source/CMSIS_RTOS/cmsis_os.h:101,
from .piolibdeps/STM32duino FreeRTOS_ID2093/src/FreeRTOS/Source/CMSIS_RTOS/cmsis_os.c:100:
.piolibdeps/STM32duino FreeRTOS_ID2093/src/portmacro.h:9:23: fatal error: stm32_def.h: No such file or directory
```

--- a/src/STM32FreeRTOS.c
+++ b/src/STM32FreeRTOS.c
@@ -11,7 +11,7 @@
  * \param[in] millis milliseconds to delay
  */
 static void delayMS(uint32_t millis) {
-  uint32_t iterations = millis * (SystemCoreClock/7000);
+  uint32_t iterations = millis * (F_CPU/7000);
   uint32_t i;
   for(i = 0; i < iterations; ++i) {
     __asm__("nop\n\t");

--- a/src/port.c
+++ b/src/port.c
@@ -3,7 +3,7 @@
  * @author  Frederic Pillon <frederic.pillon@st.com> for STMicroelectronics.
  * @brief   Include source of FreeRTOS portable layer file to match Arduino library format
  */
-#include "stm32_def.h"
+#include <Arduino.h>
 
 #ifndef __CORTEX_M
 #pragma GCC error  "no \"__CORTEX_M\" definition"

--- a/src/portmacro.h
+++ b/src/portmacro.h
@@ -6,7 +6,7 @@
 
 #ifndef _PORTMACRO_H_
 #define _PORTMACRO_H_
-#include "stm32_def.h"
+#include <Arduino.h>
 
 #ifndef __CORTEX_M
 #pragma GCC error  "no \"__CORTEX_M\" definition"

